src/components/editMeasure/details/MeasureDefinitions/MeasureDefinitions.tsx

```diff
@@ -62,13 +65,21 @@ const MeasureDefinitions = (props: MeasureDefinitionsProps) => {
     measure?.measureSet?.acls,
     measure?.measureMetaData?.draft
   );
+  const sortDefinitions = (measureDefinitions: Array<MeasureDefinition>) => {
+    if (measureDefinitions) {
+      return measureDefinitions.sort((a, b) => a.term.localeCompare(b.term));
+    } else {
+      return [];
+    }
+  };
```

Comment:
`Array.sort` function doesn't create a new array. so you don't have to return anything. I would rather change this function to a comparator for `array.sort`  
e.g.
`const sortFn = (a, b) => a.term.localeCompare(b.term);` 
and wherever needed
`measureDefinitions.sort(sortFn)`

Reply:
@adongare Changed to use the comparator, thanks!